apps/wakunode2/internal_config.nim

```diff
@@ -18,6 +21,23 @@ proc validateExtMultiAddrs*(vals: seq[string]):
     multiaddrs.add(multiaddr)
   return ok(multiaddrs)
 
+proc dnsResolve*(domain: string, conf: WakuNodeConf): Future[string] {.async} =
```

Comment:
Can we return `Result` here and either return `ok(resolved[0].host)` or `err("Could not resolve IP from DNS: empty response")`

This way the logic is encapsulated in the proc and does not leak to the caller